R/calculate.R

```diff
@@ -117,9 +117,9 @@ calculate <- function(x,
   result
 }
 
-check_if_mlr <- function(x) {
+check_if_mlr <- function(x, fn) {
   if (is_mlr(x)) {
-    stop_glue("Multiple explanatory variables are not supported in calculate().")
+    stop_glue("Multiple explanatory variables are not supported in {fn}().")
```

Comment:
Would be good to add to this message: "When working with multiple explanatory variables, use `fit()` instead."